packaging/windows/clisdk/registrykeys.wxs

```diff
@@ -10,7 +10,7 @@
       </Component>
       <Component Id="SetupRegistry_x86_RC2_Compat_Key" Directory="TARGETDIR" Win64="no">
         <RegistryKey Root="HKLM" Key="SOFTWARE\dotnet\Setup\InstalledVersions\$(var.Platform)\sdk">
-          <RegistryValue Action="write" Name="1.0.0-preview1" Type="integer" Value="1" KeyPath="yes"/>
+          <RegistryValue Action="write" Name="1.0.0-preview2" Type="integer" Value="1" KeyPath="yes"/>
```

Comment:
This is our compat reg key. This should continue to be `1.0.0-preview1` and the registry key [here](https://github.com/dotnet/cli/pull/2954/files/1033e61d57f768fe13c7412964035c41669f3f2e#diff-3b7392fa2639937bbc3db122ddaf03b7R8) will set the `preview2` value.